.circleci/config.yml

```diff
@@ -107,27 +114,48 @@ jobs:
       - run: yarn build:webpack
 
   build-e2e:
+    working_directory: ~/vets-website
     docker:
       - image: cypress/base:10.15.3
     steps:
       - checkout
       - restore_cache:
           keys:
-            - yarn-packages-{{ checksum "yarn.lock" }}
+            - yarn-packages-{{ .Branch }}-{{ checksum "yarn.lock" }}
+      - restore_cache:
+          keys:
+            - node-v1-{{ .Branch }}-{{ checksum "package.json" }}
```

Comment:
Could these be combined like this?

```
      - restore_cache:
          keys:
            - yarn-packages-{{ .Branch }}-{{ checksum "yarn.lock" }}
            - node-v1-{{ .Branch }}-{{ checksum "package.json" }}
```

Reply:
Each `restore_cache` step only restores one cached key. Multiple keys in this step serve as backups in case the first cached key fails. Here's an [example](https://circleci.com/docs/2.0/caching/#example-caching-configuration).